src/seedu/addressbook/data/AddressBook.java

```diff
@@ -79,6 +88,10 @@ private void syncTagsWithMasterList(Person person) {
     public void addPerson(Person toAdd) throws DuplicatePersonException {
         syncTagsWithMasterList(toAdd);
         allPersons.add(toAdd);
+        List<Tag> allTags = toAdd.getTags().getInternalList();
+        for(Tag tag: allTags){
+            tagHistory.addTagHistory(toAdd.getName().toString(), tag.toString(), "+");
```

Comment:
You may want to use other ways besides a magic string for the tagging prefix here.